Modules/Core/Common/src/itkPlatformMultiThreaderPosix.cxx

```diff
@@ -65,21 +64,15 @@ PlatformMultiThreader::MultipleMethodExecute()
   // will call m_MultipleMethods[NumberOfWorkUnits-1]().  When it is done,
   // it will wait for all the children to finish.
   //
-  // First, start up the m_NumberOfWorkUnits-1 processes.  Keep track
-  // of their process ids for use later in the pthread_join call
-
-  pthread_attr_t attr;
+  // First, start up the m_NumberOfWorkUnits-1 threads.  Keep track
+  // of their thread ids for use later in the pthread_join call
 
-  pthread_attr_init(&attr);
-#  ifndef __CYGWIN__
-  pthread_attr_setscope(&attr, PTHREAD_SCOPE_PROCESS);
-#  endif
   for (ThreadIdType thread_loop = 1; thread_loop < m_NumberOfWorkUnits; ++thread_loop)
   {
     m_ThreadInfoArray[thread_loop].UserData = m_MultipleData[thread_loop];
     m_ThreadInfoArray[thread_loop].NumberOfWorkUnits = m_NumberOfWorkUnits;
     int threadError = pthread_create(&(process_id[thread_loop]),
-                                     &attr,
+                                     nullptr,
```

Comment:
Just for my understanding: Is this only a style improvement, passing null instead of a default initialized `attr`? Is it still necessary to do so, given that we want to remove pthreads anyway?